.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "mongodb-odbc-driver/libmongosql"]
 	path = mongodb-odbc-driver/libmongosql
-	url = git@github.com:mongodb/libmongosql
+	url = https://github.com/nbagnard/libmongosql
```

Comment:
Seems to me that it would be better to add this branch to `mongodb/libmongosql` and use that instead of Natacha's fork?  Or it done this way because there are limitations from doing that.